Source/linting/codeanalysis.ruleset

```diff
@@ -444,7 +444,7 @@
     <Rule Id="SA1508" Action="Error" />
     <Rule Id="SA1513" Action="Error" />
     <Rule Id="SA1516" Action="Error" />
-    <Rule Id="SA1518" Action="Error" />
+    <Rule Id="SA1518" Action="None" />
```

Comment:
Can you also add to editorconfig this:
`dotnet_diagnostic.SA1518.severity = none`